packages/bundle-size/src/utils/buildFixture.js

```diff
@@ -57,10 +57,13 @@ function webpackAsync(webpackConfig) {
   return new Promise((resolve, reject) => {
     const compiler = webpack(webpackConfig);
 
-    compiler.run(err => {
+    compiler.run((err, result) => {
       if (err) {
         reject(err);
       }
+      if (result && result.hasErrors()) {
```

Comment:
is it possible to add a test for this with a fixture that should break ? you know first hand how hard this was to debug :)